src/UglyToad.PdfPig/Writer/Fonts/ToUnicodeCMapBuilder.cs

```diff
@@ -1,117 +1,106 @@
 ﻿namespace UglyToad.PdfPig.Writer.Fonts
 {
+    using Graphics.Operations;
     using System;
     using System.Collections.Generic;
     using System.IO;
-    using Graphics.Operations;
     using Tokens;
     using Util;
 
     internal static class ToUnicodeCMapBuilder
     {
-        private const string BeginToken = "begin";
-        private const string BeginCMapToken = "begincmap";
-        private const string DefToken = "def";
-        private const string DictToken = "dict";
-        private const string FindResourceToken = "findresource";
+        private static ReadOnlySpan<byte> BeginToken => "begin"u8;
```

Comment:
`static readonly`

Reply:
This is the odd syntax they chose for statically initialized data, which is a method rather than a property (so we can't add readonly). It points to a fixed position in the assembly, which is readonly and immutable. It looks like it's allocating, but it's not.

https://vcsjones.dev/csharp-readonly-span-bytes-static/

NOTE, this only works for bytes on .NET versions before .NET 7.0.